apps/api/src/app/events/services/workflow.queue.service.ts

```diff
@@ -7,6 +7,10 @@ import { QueueNextJobCommand } from '../usecases/queue-next-job/queue-next-job.c
 import { JobEntity, JobRepository, JobStatusEnum } from '@novu/dal';
 import { ChannelTypeEnum, DigestUnitEnum } from '@novu/shared';
 
+interface IJobEntityExtended extends JobEntity {
+  presend?: boolean;
```

Comment:
```suggestion
  present?: boolean;
```

Maybe `present` better reflects this was in past?